### PR TITLE
I fixed the outstanding issues for the lidarr website.

### PR DESCRIPTION
**Pull request by dnfm**
Feel free to use/modify all or none or some of them as you see fit. :)

### CODE REVIEW

**Comment by Qstick** [comment]
Nice thanks, we will take a look

**Comment by majora2007** [comment]
Looks pretty good. Good job. I agree, after looking at our original site we threw together, we are in need of an update.

File: index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <link rel="stylesheet" type="text/css" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" />
     <link rel="stylesheet" type="text/css" href="css/bootstrap-overrides.css" />
     <link rel="stylesheet" type="text/css" href="//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css" />
```
Comment (Qstick) [inline index.html:11]: Do we need old FA reference still?

**Comment by Qstick** [comment]
Added with c5df569a14390593cd42d796b58d4cd07bfed183

**Pull request closed without merging**